docs/maya/glue.md

```diff
@@ -0,0 +1,145 @@
+
+### Node Attributes
+| Name | Type | Default | Animatable | Description |
+| :--- | :--- | :------ | :--------- | :---------- |
+| **Envelope** | Float | 1.0 | ✓ | Specifies the deformation scale factor. Has a range of \[0.0, 1.0\]. The upper and lower limits are soft, values can be set in a range of \[-2.0, 2.0\]|
+
+### Debug Attributes
+| Name | Type | Default | Animatable | Description |
+| :--- | :--- | :------ | :--------- | :---------- |
+| **Debug**       | Boolean      | False                | ✓ | Enable or Disable the debug functionalities in the viewport for the AdnGlue node. |
+| **Feature**     | Enumerator   | Distance Constraints | ✓ | A list of debuggable features for this node.<ul><li>Distance Constraints: Draw *Distance Constraint* connections representing the constrained pair of vertices in the simulated mesh.</li><li>Shape Preservation: Draw *Shape Preservation* connections between the vertices adjacent to the vertices with this constraint.</li><li>Glue Constraints: Draw *Glue* connections between every vertex and its closest point on the closest neighbor surface.</li></ul> |
+| **Width Scale** | Float        | 3.0                  | ✓ | Modifies the width of all lines. |
+| **Color**       | Color Picker | Red                  | ✓ | Selects the line color from a color wheel. Its saturation can be modified using the slider. |
+
+## Attribute Editor Template
+
+<figure markdown>
+  ![glue editor](images/glue_attribute_editor_00.png) 
+  <figcaption><b>Figure 1</b>: AdnGlue Attribute Editor.</figcaption>
+</figure>
+
+## Paintable Weights
+
+In order to provide more artistic control, some key parameters of the AdnGlue solver are exposed as paintable attributes in the node. The Maya paint tool must be used to paint those parameters to ensure that the values satisfy the solver requirements.
+
+| Name | Default | Description |
+| :--- | :------ | :---------- |
+| **Compression Resistance**      | 1.0 | Force to correct the edge lengths if the current length is smaller than the rest length. A higher value represents higher correction.<ul><li>*Tip*: To optimize the painting of the weight, flood it to 1.0 as a starting point and tweak some areas later on.</li><li>*Tip*: Reducing the value of the weight in some areas will contribute to reduce wrinkling effect.</li></ul> |
+| **Glue Resistance**      | 1.0 | Force to preserve the distance to the closest point on the closest neighbor surface. A higher value represents higher correction.<ul><li>*Tip*: Paint a value of 0.0 in those areas where the gluing effect is not needed and it will increase the performance.</li></ul> |
+| **Masses**                      | 1.0 | Multiplier to the individual mass values per vertex. |
+| **Max Glue Distance Multiplier** | 1.0 | Multiplier to the individual values of the max glue distance per vertex. |
```

Comment:
If I'm not mistaken for "Max Glue Distance Multiplier" we directly skip the constraint creation if the weight is set to 0.0. Should we also add something like "Paint a value of 0.0 in those areas where the gluing effect is not needed and it will increase the performance."?